src/js/kitchensink.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component }  from "react";
+import ReactDOM from 'react-dom';
+import '../scss/app.scss';
+import '../scss/kitchensink.scss';
+import Chat from "./components/Chat/Chat";
+import MainNav from "./components/MainNav/MainNav";
+import GroupNav from "./components/GroupNav/GroupNav";
+import AddServerButton from "./components/AddServerButton/AddServerButton";
+import MainNavButton from "./components/MainNavButton/MainNavButton";
+import Config from "./utilities/Config";
+import { configuration } from "./config/kitchensink";
+import { GlobalContextWrapper } from "./contexts/GlobalContextWrapper";
+
+class KitchenSink extends Component {
+  render () {
+    return (
+      <GlobalContextWrapper Config={ new Config(configuration) } >
+        <div className="dishes">
+          <MainNavButton id={'test'} title={'Test Button'} iconClassName={''} imageSrc={''} channelExtraClassNames={['nav-channel-blue']} contentExtraClassNames={['']} sort={''} />
```

Comment:
This is actually a pretty effective way of showing all the available props. Nice one